search-app/docs/api.md

```diff
@@ -2804,7 +2804,7 @@ Several fields including science keywords, data centers, platforms, instruments,
 
 #### <a name="facets-v2-response-format"></a> Version 2 Facets Response Format
```

Comment:
I think you also added support for searching collections by hierarchical platforms parameter. This needs to be added to the api doc for completeness.

Reply:
Need an example of how the new search works and how it differs from the existing platform search.